generators/generator-base-private.js

```diff
@@ -315,34 +314,6 @@ module.exports = class extends Generator {
         }
     }
 
-    /**
-     * insight
-     *
-     * @param trackingCode
-     * @param packageName
-     * @param packageVersion
-     * @returns {Insight}
-     */
-    insight(trackingCode = 'UA-46075199-2', packageName = packagejs.name, packageVersion = packagejs.version) {
```

Comment:
I wouldn't completely remove this since it will reduce stats a lot as the linking is really opt-in and multistep(due to the login part) and I don't think most people will do it.

@jdubois WDYT? IMO we should keep existing as well atleast for a while

Reply:
normally (I didn't check the code), the user would send the data anonymously like today. Then, if he chooses to, he can also login (that step is optional), so he can have more features in JHipster Online - in that case, his data will be linked to his account, and he will also be able to delete it

Reply:
see my previous comment, I guess it would be better user experience to automate it more

Reply:
yes we should keep the existing Insight code - I need to check in the code but that's what @agaspardcilia told me he did